sdk/src/types/block/signature/mod.rs

```diff
@@ -24,7 +26,13 @@ use crate::types::block::Error;
 pub enum Signature {
     /// An Ed25519 signature.
     #[packable(tag = Ed25519Signature::KIND)]
-    Ed25519(Ed25519Signature),
+    Ed25519(Box<Ed25519Signature>),
```

Comment:
Whats the box for?

Reply:
The underlying type is no longer heap allocated, so this is needed for the enum.